tests/check_framework/tests.py

```diff
@@ -305,3 +305,99 @@ class ModelWithDescriptorCalledCheck(models.Model):
 class ChecksRunDuringTests(SimpleTestCase):
     def test_registered_check_did_run(self):
         self.assertTrue(my_check.did_run)
+
+
+class CheckDuplicateDBTableTests(SimpleTestCase):
```

Comment:
Please put this in a new file called `test_model_checks.py`. We can move the rest of the model check tests in a separate PR. (Or you can do the move of existing tests first and then rebase this PR.)